packages/flagship/src/lib/fs.ts

```diff
@@ -65,8 +70,12 @@ export function update(path: string, oldText: string | RegExp, newText: string):
  * @param {string} keyword The keyword to search for in the file
  * @returns {boolean} True if the keyword exists in the file.
  */
-export function doesKeywordExist(path: string, keyword: string): boolean {
+export function doesKeywordExist(path: string, keyword: string | RegExp): boolean {
   // TODO: This should use a streaming buffer
   const fileContent = readFileSync(path, { encoding: 'utf8' });
-  return fileContent.indexOf(keyword) > -1;
+  if (typeof(keyword) === 'string') {
```

Comment:
This should just be `if (typeof keyword === 'string') {`

Reply:
Tagging a commit on @alexrisch behalf. Coming right up @bweissbart.